site/src/components/Dashboard/Navbar/NavbarView.tsx

```diff
@@ -296,6 +308,8 @@ const ProxyMenu: FC<ProxyMenuProps> = ({ proxyContextValue }) => {
           }}
         >
           <h4
+            autoFocus
+            tabIndex={-1}
```

Comment:
Safety nets to make extra sure that the header receives focus when the menu opens. 99% sure `tabIndex` shouldn't be necessary, but it's just a precaution to make sure that it's never directly tab-able